.env

```diff
@@ -4,5 +4,5 @@ NODE_VERSION=16
 THECODINGMACHINE_VERSION=v4
 WORDPRESS_VERSION=6.5.2
 WORDPRESS_DEV_VERSION=6.5.2
-WOOCOMMERCE_VERSION=8.7.0
+WOOCOMMERCE_VERSION=9.2.2
```

Comment:
can you change https://github.com/storekeeper-company/storekeeper-woocommerce-b2c/blob/master/Readme.txt#L5 as well?